phoenix-core/src/main/java/org/apache/phoenix/schema/PColumnImpl.java

```diff
@@ -156,14 +209,14 @@ public String toString() {
         return (familyName == null ? "" : familyName.toString() + QueryConstants.NAME_SEPARATOR) + name.toString();
     }
     
-    @Override
-    public int hashCode() {
-        final int prime = 31;
-        int result = 1;
-        result = prime * result + ((familyName == null) ? 0 : familyName.hashCode());
-        result = prime * result + ((name == null) ? 0 : name.hashCode());
-        return result;
-    }
+	@Override
+	public int hashCode() {
+	    final int prime = 31;
+	    int result = 1;
+	    result = prime * result + ((familyName == null) ? 0 : familyName.hashCode());
+	    result = prime * result + ((name == null) ? 0 : name.hashCode());
+	    return result;
+	}
```

Comment:
Minor nit - various formatting issues. Not sure if there are tabs now or if the indenting was wrong before.

Reply:
Fixed